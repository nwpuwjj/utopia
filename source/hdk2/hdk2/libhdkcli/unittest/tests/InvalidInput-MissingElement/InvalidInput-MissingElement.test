#
# InvalidInput-MissingParameter.test
#    Verify libhdkcli handles an invalid input struct which is missing a required parameter.
#

method: http://cisco.com/Unittest/SimpleUnittestMethod
