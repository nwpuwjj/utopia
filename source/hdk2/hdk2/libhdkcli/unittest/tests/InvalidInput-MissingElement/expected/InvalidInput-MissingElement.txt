
******* Client Result ******

      SOAP action: http://cisco.com/Unittest/SimpleUnittestMethod
        HTTP host: http://localhost:8080
Network Object ID: (null)
    HTTP username: (null)
    HTTP password: (null)
          timeout: 0
       HNAP input: InvalidInput-MissingElement.input.xml

HNAP call failed with error RequestInvalid (1)
malloc_interposer.c - ************************************************************
malloc_interposer.c - exit: malloc called 63 times
malloc_interposer.c - exit: free called 63 times
malloc_interposer.c - exit: malloc current = 0
malloc_interposer.c - exit: malloc total = 11888
malloc_interposer.c - exit: malloc max = 11104
malloc_interposer.c - ************************************************************
